src/renderer/vt/paint.cpp

```diff
@@ -558,7 +555,7 @@ using namespace Microsoft::Console::Types;
         {
             _deferredCursorPos = { _lastText.X + sNumSpaces, _lastText.Y };
         }
-        else if (numSpaces > 0)
+        else if (removeSpaces && numSpaces > 0)

```

Comment:
If we're removing spaces, and there were spaces, print them? I am confus

Reply:
Man so am I, I have no idea why that fixed the test and didn't break any of the other ones. It was literally the first thing I guessed, looks like I didn't actually add a comment